torch/_torch_docs.py

```diff
@@ -8309,10 +8309,14 @@ def merge_dicts(*dicts):
 
 .. note:: The `S` tensor can only be used to compute gradients if :attr:`compute_uv` is True.
 
-
 .. note:: With the complex-valued input the backward operation works correctly only
           for gauge invariant loss functions. Please look at `Gauge problem in AD`_ for more details.
 
+.. note:: Since `U` and `V` of an SVD is not unique, each vector can be multiplied by
```

Comment:
@IvanYashchuk and @antocuni, would you take a look at this in particular?

Also, we should not use the term "backend" or suggest that NumPy might be used in PyTorch's computation of torch.{linalg.}svd. Maybe something like, "Different platforms, like NumPy, or inputs on different device types, may produce different `U` and `V` tensors."?

Reply:
It was suggested to be documented https://github.com/pytorch/pytorch/pull/48436#discussion_r554984923, though I don't think these details would be really necessary to users.

I'll first modify the doc as you suggested.

Reply:
"each vector can be multiplied by" there is no vector mentioned in the main description, so better to avoid this term, maybe use "columns of matrices".
"`e^{i phi}`" -> ":math:`e^{i \phi}`"

Many people with applied math or physics background may already know that the left and right singular vectors are unique up to multiplication by a unit-phase factor (or sign for reals) if singular values are not degenerate, that's Wikipedia.
Here the text, I assume, is for programmers, deep learning practitioners, and other researchers and the main goal is to prevent bug reports "numbers are different between PyTorch / NumPy / etc."
Maybe omitting some details is better, like "`U` and `V` tensors are not mathematically unique. Different platforms, like NumPy, or inputs on different device types, may produce different `U` and `V` tensors."

Reply:
I think it's a good idea to clearly specify in the docs that the output can be different than numpy's and that it's not a bug.
Consider the use case in which someone has a test suite which compares svd's output with a hard-coded result, and then tries to migrate the code from numpy to pytorch: knowing why the test fails is an extremely valuable information in that scenario.